jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/ec2/VMsWithSeverityVulnerabilityRule.java

```diff
@@ -105,10 +109,9 @@ public PolicyResult execute(Map<String, String> ruleParam, Map<String, String> r
                 JsonObject firstVulnerability = vulnerabilityInfoList.get(0);
                 Map<String, Object> issueDetails = buildVMIssueDetails(firstVulnerability);
                 fillMoreInfo(vulnerabilityInfoList, issueDetails);
-                JsonObject pluginDetails = firstVulnerability.get("plugin").getAsJsonObject();
 
                 Annotation annotation = Annotation.buildAnnotation(ruleParam, Annotation.Type.ISSUE);
-                annotation.put(PacmanSdkConstants.DESCRIPTION, pluginDetails.get(DESCRIPTION_JSON_FIELD).getAsString());
+                annotation.put(PacmanSdkConstants.DESCRIPTION, "EC2 instance with " + severity + " vulnerability found");
```

Comment:
I think we need to get rid of EC2 and replace with VM or Virtual Machine? When we implement additional clouds, this rule will still stay generic.

Aqua will need to change.